src/main/java/net/mcreator/ui/laf/MCreatorTheme.java

```diff
@@ -263,9 +263,11 @@ protected void initMCreatorThemeColors(UIDefaults table) {
 		table.put("RadioButton.icon", new RadioButtonIcon());
 		table.put("RadioButtonMenuItem.icon", new RadioButtonIcon());
 
+		table.put("TabbedPane.foreground", new ColorUIResource(colorScheme.getForegroundColor()));

```

Comment:
This is needed because if `(fg instanceof UIResource)` is false where `fg` is the foreground color of a `JList`, its UI class instance would not respect the value of `"TabbedPane.selectedForeground"` property and use the usual foreground color in its place.

Reply:
Not sure I understood this explanation

Reply:
In short, without wrapping this particular color into a `ColorUIResource`, the text of the selected tab does not become black.